app/views/users/terms.html.erb

```diff
@@ -28,27 +30,36 @@
     <%= render :partial => "terms" %>
   </div>
 
+  <div class="deemphasize">
+    <p id="contributorGuidance">
+      <%= raw t '.guidance',
+                :summary => 'https://www.osmfoundation.org/wiki/License/Contributor_Terms_Summary',
+                :translations => 'https://www.osmfoundation.org/wiki/License/Contributor_Terms/Informal_Translations' %>
+    </p>
+  </div>
+
+  <label class="standard-label">
+    <%= t 'layouts.tou' %>
+  </label>
+  <p class="deemphasize"><%= t '.tou_explain' %></p>
   <div class="form-row">
-    <label for="user_consider_pd">
-      <%= check_box('user', 'consider_pd') %>
-      <%= t '.consider_pd' %>
+    <label for="read_tou">
+      <%= check_box_tag 'read_tou',1,false %>
+      <%= t '.read_tou_html', :tou_link => link_to(t('layouts.tou'), "https://wiki.osmfoundation.org/wiki/Terms_of_Use", :target => :new) %>
     </label>
-    <span class="minorNote">(<%= link_to(t('.consider_pd_why'), t('.consider_pd_why_url'), :target => :new)%>)</span>
 
     <%= hidden_field_tag('referer', h(params[:referer])) unless params[:referer].nil? %>
 
     <div class="buttons form-row inner20 clearfix">
-      <p class="deemphasize"><%= t '.read and accept' %></p>
-      <%= submit_tag(t('.agree'), :name => "agree", :id => "agree") %>
+      <%= submit_tag(t('.agree'), :name => "agree", :id => "agree", disabled: true)  %>
```

Comment:
Please use the `=>` syntax for hashes.

Reply:
Fixed in https://github.com/openstreetmap/openstreetmap-website/pull/2028/commits/5dfbc92ef60112c55462bab75898c25d31d6bc84